### DISCUSSION TITLE
How to bundle local pacakge

### DISCUSSION THREAD

**Discussion by JardarTemalogic** [webpack/webpack#16483] [Q&A]
Hi,

I have a local package in a different folder (using npm link). When building this for deployment I need to bundle this package since this will not be available on the npm repository. How do I configure webpack to bundle this local package correctly.

**Comment by alexander-akait** [comment]
Just set required files to bundle

**Reply by JardarTemalogic** [reply]
Do you have an example config?

**Reply by alexander-akait** [reply]
There are a lot of configuration examples in internet, I don't know your project so I can't say, you can search them on stackoverflow and github

**Reply by JardarTemalogic** [reply]
There is probably something I do not understand with webpack. But I have looked through quite a bit of the options that can be configured in webpack.config.js and have not been able to find out how to do this.

Here is my setup:

Package A
* Node
* References Package B with npm link

Package B
* Node

Package A will be run in a docker container, thus will not be able to download the Package B from the npm repository. Which means that this specific package need to be bundled by webpack.
Below is my webpack.config.json

```
module.exports = {
    entry: [
        './src/main.ts',
    ],
    optimization: {
        minimize: false,
    },
    target: 'node',
    mode,
    devtool,
    externals: [
        nodeExternals({}),
    ],
    module: {
        rules: [
            {
                test: /\.tsx?$/,
                loader: 'ts-loader',
                options: {
                    transpileOnly: true
                },
                 exclude: '/node_modules/',
            }
        ],
    },
    resolve: {
        extensions: ['.tsx', '.ts', '.js'],
    },
    plugins: [
        new CopyPlugin({
            patterns: [
                { from: "package.json", to: "." },
                { from: "yarn.lock", to: "." },
            ],
        }),
    ],
    output: {
        filename: 'bundle.js',
        path: path.resolve(__dirname, 'dist'),
    },
};


```
